ci/jenkins/test.sh

```diff
@@ -402,6 +403,8 @@ function  build_and_deliver_antrea_windows_and_linux_images {
     ${PRINT_DOCKER_STATUS}
     export_govc_env_var
     # Enable verbose log for troubleshooting.
+    curl -OL http://build-squid.vcfd.broadcom.net/build/mts/release/sb-82828805/publish/windows-advanced/manifests/${WINDOWS_YAML_NAME}.yml
```

Comment:
Do not expose Broadcom internal URLs on public Internet.

Reply:
We won't use it after the testbed is restored. cc @wenyingd